modules/op-resolver-git-config-anon.adoc

```diff
@@ -0,0 +1,37 @@
+// This module is included in the following assembly:
+//
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+
+:_content-type: PROCEDURE
+[id="resolver-git-config-anon_{context}"]
+= Configuring the git resolver for anonymous Git cloning
+
+If you want to use anonymous Git cloning, you can optionally configure the default Git revision, fetch timeout, and default repository URL for pulling remote pipelines and tasks from a Git repository.
```

Comment:
```suggestion
If you want to use anonymous Git cloning, you can configure the default Git revision, fetch timeout, and default repository URL for pulling remote pipelines and tasks from a Git repository.
```
again, "can" rather than "must" implies it's optional